understanding/consistent-help.html

```diff
@@ -53,322 +52,225 @@
-                     a few interactions (e.g., displayed in the header, or via a menu). In addition, for
-                     some users with disabilities, struggling to complete a task on a site may cause additional
-                     cognitive challenges when searching for help within the site.
+                  <p>インターネット検索を使用して組織の連絡先情報を見つけるなど、サイトを越えた自己解決の方法は難しすぎる場合がある。さらに、情報が数回の対話内で一貫して存在しない場合 (例えば、ヘッダーに表示される、メニュー経由)、利用者の障害により、利用可能なヘルプ (「お問い合わせ」リンク、電話番号、サポートページなど) を見つけることがより困難になる可能性がある。加えて、障害のある一部の利用者によっては、サイト上で作業を完了するのに苦労することで、サイト内でヘルプを検索するときに、さらなる認知的な問題が発生する可能性がある。
                   </p>
                   
                   
-                  <p>When a user is quickly able to find help, they are able to complete the task even
-                     if they encounter challenges.
+                  <p>利用者がすぐにヘルプを見つけることができれば、たとえ困難に直面したとしても作業を完了することができる。
                   </p>
                   
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
+               <h2>メリット</h2>
                <ul>
                   
-                  <li>People who may have difficulty locating help are more likely to find it when it is
-                     consistently located.
+                  <li>ヘルプを見つけるのが困難な人々でも、ヘルプが一貫して置かれるとき、ヘルプを見つける可能性が高くなる。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>On-line job application: Some of the application questions may be hard for new job
-                     seekers to understand even after reading the contextual help. For example, the form
-                     may request their identification number, but they may have several and not know which
-                     one to enter. Consistently located contact information will enable them to use phone
-                     or email so they can get an answer to their question.
+                  <li>オンラインでの求人応募: 応募に関する質問の中には、コンテキストに沿ったヘルプを読んだ後でも、新規求職者にとって理解するのが難しいものがある。例えば、フォームで識別番号を要求するが、複数の識別番号があり、どれを入力すればよいかわからない場合がある。連絡先情報を一貫して配置することで、電話又は電子メールを使用して質問に対する回答を得ることができる。
                   </li>
                   
-                  <li>Medical appointment scheduling form: When the service a patient is trying to book
-                     is not easily findable within the interface, they may need human help. A consistently
-                     located messaging option (chat client) enables them to quickly interact with a staff
-                     person that can help, without requiring them to manage a second interface.
+                  <li>診察予約スケジュールフォーム: 患者が予約しようとしているサービスがインターフェース内で簡単に見つけられない場合、人間の助けが必要になる場合がある。一貫して配置されているメッセージングオプション (チャットクライアント) により、別のインターフェースを管理する必要がなく、サポートできるスタッフと素早く対話できる。
```

Comment:
「without requiring them to manage a second interface.」の manage は、管理というよりは (別のインターフェースを用いて) なんとかする、という感じなのかなと思いました。とした場合、以下は、いかがでしょうか？

```suggestion
                  <li>診察予約スケジュールフォーム: 患者が予約しようとしているサービスがインターフェース内で簡単に見つけられない場合、人間の助けが必要になる場合がある。一貫して配置されているメッセージングオプション (チャットクライアント) により、別のインターフェースでどうにか解決しようとする必要がなく、サポートできるスタッフと素早く対話できる。
```

Reply:
https://eow.alc.co.jp/search?q=manage に
> 〔道具や武器などを巧みに〕使う、扱う、操る

とあるので、「操作する」とするのはどうでしょうか

```suggestion
                  <li>診察予約スケジュールフォーム: 患者が予約しようとしているサービスがインターフェース内で簡単に見つけられない場合、人間の助けが必要になる場合がある。一貫して配置されているメッセージングオプション (チャットクライアント) により、別のインターフェースを操作する必要がなく、サポートできるスタッフと素早く対話できる。
```

Reply:
なるほどです。よいと思います！